src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -170,6 +170,7 @@ public boolean equals(Object obj) {
     @Override
     public synchronized void addCard(Card card) throws DuplicateCardException {
         addressBook.addCard(card);
+        updateFilteredCardList();
```

Comment:
I think this is already called in `indicateAddressBookChanged()` so no need to call it here